specification/07-theory_operations.adoc

```diff
@@ -525,15 +525,62 @@ host supervisor domain manager may directly assign a TDI to a TVM, through the
 `COVH` interface. This is a four steps process:
 
 1. The host supervisor domain manager initiates the interface assignment flow by
-   having the TSM move the TDI into the TDISP `CONFIG_LOCKED` state.
-2. The TVM verifies and accepts the locked TDI into its TCB.
-3. The TVM asks the TSM to move the TDI to the TDISP `RUN` state.
+   having the TSM move the TDI into the TDISP `CONFIG_LOCKED` state. This is
+   achieved through the `sbi_covh_add_interface()` `COVH` ABI.
+2. The TVM xref:_tdi_acceptation[verifies and accepts the locked TDI] into its
+   TCB.
+3. The TVM asks the TSM to move the TDI to the TDISP `RUN` state, by calling the
+   `sbi_covg_run_interface()` `COVG` ABI.
 4. The TVM verifies that the TDI is in the TDISP `RUN` state and starts
-   using it.
-
-The next section describe in more details what step 2 covers, i.e. which steps
-a TVM should follow in order to be able to verify a locked TDI and accept or
-reject it into its TCB.
+   using it. This verification is provided by the
+   `sbi_covg_get_interface_state()` `COVG` ABI.
+
+The next two sections respectively give a detailed description of the overall
+assignment process and one if its most critical steps: the TVM decision of
+accepting or rejectin the TDI into its TCB.
+
+==== Assignment Flow
+
+Assigning an interface to a TVM goes through the following steps:
+
+1. The host supervisor domain manager calls the `sbi_covh_add_interface()`
+   `COVH` function to initiate the TDI assignment. The TSM rejects that request
+   if a secured SPDM session is not established with the DSM or if a stream IDE
+   link is not set up.
+2. The TSM generates and sends a TDISP `LOCK_INTERFACE_REQUEST` message to the
```

Comment:
The overall flow is good.

Question: Do we want to list all TDISP sequence?

I think the TSM needs to send `GET_TDISP_VERSION` and `GET_TDISP_CAPABILITIES`, before `LOCK_INTERFACE_REQUEST`.

Reply:
> The overall flow is good.
> 
> Question: Do we want to list all TDISP sequence?

I think we should, yes.

> I think the TSM needs to send `GET_TDISP_VERSION` and `GET_TDISP_CAPABILITIES`, before `LOCK_INTERFACE_REQUEST`.

I added those now.